src/calendar-twig/composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": "~8.2.0 || ~8.3.0 || ~8.4.0",
         "aeon-php/calendar": "~1.0",
-        "twig/twig": "~2.10||~3"
+        "twig/twig": "^3.19"
```

Comment:
same goes here, if there is no maintenance issue involved, supporting more versions of twig makes this lib available to wider audience

Reply:
oh right, yea since it was deprecated 2023 I think it's time 👍